openbas-front/src/admin/components/common/form/TextFieldAskAI.tsx

```diff
@@ -194,22 +187,22 @@ const TextFieldAskAI: FunctionComponent<TextFieldAskAiProps> = ({
               {t('Generate an article')}
             </MenuItem>
           )}
-          <MenuItem onClick={() => handleAskAi('spelling')}>
+          <MenuItem onClick={() => handleAskAi('spelling')} disabled={currentValue.length === 0}>
```

Comment:
It would be nice to add a popover to explain to the user why it is grayed out. What do you think ?

Reply:
Sure, but I have a doubt because when you try to generate some text, you get this message: '**Input (describe what you want).**' So maybe that is enough?

Reply:
When did you get this message? I miss it I think

Reply:
Ah, sorry! You're right. I could add a tooltip to explain the status of the button :)